### PR TITLE
Atlas fts

### PR DESCRIPTION
**Pull request by kenwalger**
Atlas Full-Text Search Post

http://docs-devhub-staging.s3-website-us-east-1.amazonaws.com/8ba64a9/devhub/docsworker-xlarge/atlas-fts/how-to/build-movie-search-engine-using-atlas-full-text-search-in-10-minutes

https://workerpool-boxgs.mongodbstitch.com/pages/job.html?jobId=5e599c2d0ec9481dfdaa9fa0

### CODE REVIEW

File: source/how-to/build-movie-search-engine-using-atlas-full-text-search-in-10-minutes.txt [hunk 1]
```diff
@@ -0,0 +1,587 @@
...
+      :alt: Break out the popcorn
+      :align: center
+
+   To **Get Started**, we will need:
+
+   1. A free tier (M0) cluster on MongoDB Atlas. Click here to sign up for
+      an account and deploy your free cluster on your preferred cloud
+      provider and region.
+
+   .. container:: callout
```
Comment (schmalliso) [inline source/how-to/build-movie-search-engine-using-atlas-full-text-search-in-10-minutes.txt]: swaped `.. container:: callout` for `.. blockquote::` and then indent this three spaces under the 1. paragraph

File: source/how-to/build-movie-search-engine-using-atlas-full-text-search-in-10-minutes.txt [hunk 2]
```diff
@@ -0,0 +1,587 @@
...
+      Want to try our Full-Text Search on a cluster of your own?
+      ``YouOldMaid100`` for $100 of Atlas credit. For information on
+      redeeming Atlas credit, see `Atlas
+      Billing <https://www.google.com/url?q=https://www.mongodb.com/cloud/atlas/signup?form-type%3Dfree&sa=D&ust=1579301554143000&usg=AFQjCNHI71SacwhKWJLcHMUa9RvY5bAoKw>`_.
+
+   2. The Atlas sample dataset loaded into your cluster. You can load the
+      sample dataset by clicking the ellipse button and **Load Sample
+      Dataset**.
+
+   .. image:: /images/atlas-fts/load_sample_data.png
```
Comment (schmalliso) [inline source/how-to/build-movie-search-engine-using-atlas-full-text-search-in-10-minutes.txt]: indented these two as well so they're in the `2` content

File: source/how-to/build-movie-search-engine-using-atlas-full-text-search-in-10-minutes.txt [hunk 3]
```diff
@@ -0,0 +1,587 @@
...
+
+   *You do not have to use Compass for this stage, but I really love the
+   easy-to-use UI Compass has to offer. Plus the ability to preview the
+   results by stage makes troubleshooting a snap! For more on Compass’
+   Aggregation Pipeline Builder, check out
+   this*\ `blog <https://www.mongodb.com/blog/post/introducing-the-aggregation-pipeline-builder-in-mongodb-compass>`_\ *.*
+
+   Navigate to the Aggregations tab in the ``sample_mflix.movies`` collection:
+
+   .. image:: /images/atlas-fts/aggregations_tab.png
```
Comment (schmalliso) [inline source/how-to/build-movie-search-engine-using-atlas-full-text-search-in-10-minutes.txt]: fixed path to be aggregations-tab.png

File: source/how-to/build-movie-search-engine-using-atlas-full-text-search-in-10-minutes.txt [hunk 4]
```diff
@@ -0,0 +1,587 @@
...
+
+   Stage 2: $project
+   ~~~~~~~~~~~~~~~~~
+
+   .. image:: /images/atlas-fts/project.png
+      :alt: "Project dialog"
+      :align: center
+
+   We use
+   **:math:`project** to get back only the fields we will use in our movie search application. We also use the **`\ meta**
```
Comment (schmalliso) [inline source/how-to/build-movie-search-engine-using-atlas-full-text-search-in-10-minutes.txt]: :math:`project` will link to something weird in Python, I'd swap it to``$project``.

File: source/how-to/build-movie-search-engine-using-atlas-full-text-search-in-10-minutes.txt [hunk 5]
```diff
@@ -0,0 +1,587 @@
...
+   operator to surface each document’s **searchScore** and
```
Comment (schmalliso) [inline source/how-to/build-movie-search-engine-using-atlas-full-text-search-in-10-minutes.txt]: all of these `**field**` instances could probably be 

```
``field``
```
instead to be monospaced and code-like

Preview:
Screenshot 2020-03-02 15 27 51

File: source/how-to/build-movie-search-engine-using-atlas-full-text-search-in-10-minutes.txt (hunk 4 above)
Comment (schmalliso) [inline source/how-to/build-movie-search-engine-using-atlas-full-text-search-in-10-minutes.txt]: These multiline bold/italic blocks may render weirdly, I'm not sure about how that will behave, but currently it looks fine.

File: source/how-to/build-movie-search-engine-using-atlas-full-text-search-in-10-minutes.txt [hunk 6]
```diff
@@ -0,0 +1,587 @@
...
+   .. image:: /images/atlas-fts/query-results.png
+      :alt: Results of query
+      :align: center
+
+   If this is working, congrats! We are almost done! Make sure to **SAVE**
+   and deploy the service by clicking **REVIEW & DEPLOY CHANGES** at the
+   top of the screen.
+
+   Use the API
+   ```````````
```
Comment (schmalliso) [inline source/how-to/build-movie-search-engine-using-atlas-full-text-search-in-10-minutes.txt]: This is the wrong heading level - swapped the `` for ~~ to be consistent with the earlier headings to quell the parser error

**Comment by schmalliso** [comment]
Oh, I also rebased, so you'll need to pull the branch down again if you want to make any tweaks.

**Comment by kenwalger** [comment]
Merge away. No toe stomping involved.

**Comment by schmalliso** [comment]
Done! Merged.

**Pull request merged**